public/books/ZG/songs.json

```
@@ -1,1991 +1,2991 @@
 {
 	"1": {
 		"title": "U tamnoj noći",
-		"notes": ["E4", "C4", "G3", "C3"]
+		"notes": ["E4", "C4", "G3", "C3"],
+		"page_num": 0,
+		"starts_at_top": true
```

Could you remove the page_num and starts_at_top keys for the production songs.json
```suggestion
		"notes": ["E4", "C4", "G3", "C3"]
```

---

> Could you also remove the page_num and starts_at_top keys for the production songs.json

removed